nahusenayElias/Team_4
# DRD-130: create content in Drupal for filled contact form segment

## Related ticket
[DRD-130](https://edu-team4-react24k.atlassian.net/jira/software/projects/DRD/boards/1?selectedIssue=DRD-130)

## In this PR
- added a new text with image paragraph for filled contact form segment

## Testing instructions
- checkout branch
- lando db-import DRD-130.sql.gz
- Check front page, there should be a new paragraph if you are in the filled contact form segment
<img width="1211" alt="Screenshot 2024-12-19 at 9 46 40" src="https://github.com/user-attachments/assets/3f0d9ef0-fee7-43a1-98eb-c428bb868cf5" />


[DRD-130]: https://edu-team4-react24k.atlassian.net/browse/DRD-130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
